.editorconfig

```diff
@@ -0,0 +1,8 @@
+[*]
+insert_final_newline=true
+indent_style=space
+indent_size=2
+
+[{*.html, *.hbs}]
+indent_size=4
```

Comment:
thoughts on having this 2 as well?

Reply:
it's more messy with 2 spaces because 1 is used for '<'
all current codebase is with 4 spaces already